### ISSUE TITLE
VQ2D evaluation: No matching checkpoint file found

### ISSUE THREAD

**Issue by alexandrehirschX** [EGO4D/episodic-memory#24]
Running evaluate_vq2d.py yields the following error:

**Comment by srama2512** [comment]
@alexandrehirschX  - Have you updated `<CHECKPOINT_ID>` in `evaluate_vq2d.py`?

**Comment by alexandrehirschX** [comment]
@srama2512 - Yes, I updated the path to where I have downloaded the models:
`model.config_path="$VQ2D_ROOT/data/vq2d_models/vq2d_models/siam_rcnn_residual/config_0.yaml" \
model.checkpoint_path="$VQ2D_ROOT/data/vq2d_models/vq2d_models/siam_rcnn_residual/model_0.pth" \`
I renamed the model and config files to resemble the "model_<CHECKPOINT_ID>.pth" name.

**Comment by srama2512** [comment]
My bad. What I meant was, can you just try replacing `model_<CHECKPOINT_ID>.pth` with `model_0.pth` in https://github.com/EGO4D/episodic-memory/blob/main/VQ2D/scripts/evaluate_vq.sh?

**Comment by alexandrehirschX** [comment]
Yes that's what I did, though the evaluate_vq.sh script that I am running is the one that has been copied to VQ2D as per the following instructions (the VQ2D_ROOT is my EXPT_ROOT).

![Screenshot from 2022-06-02 10-08-48](https://user-images.githubusercontent.com/51078543/171533087-7544c386-04e1-47fb-a23a-ec1740d92aa4.png)

**Comment by srama2512** [comment]
That's strange. Something seems off. @alexandrehirschX  - can you send me your copy of `evaluate_vq.sh` and your current directory structure? That is, where is `evaluate_vq.sh` is located and the contents of EXPT_ROOT?

**Comment by alexandrehirschX** [comment]
Hi, 

Here attached are the requested files. 

Best, 

Alexandre Hirsch

**Comment by srama2512** [comment]
@alexandrehirschX  - Can you also send the `evaluate_vq.sh` script that you are using? I suspect that the paths are not set correctly.

**Comment by alexandrehirschX** [comment]
The problem was in vq2d/config, the KYS tracker file path was not correct. I don't know why that happened.

**Comment by srama2512** [comment]
@alexandrehirschX  - if you feel there are changes that can make the code / doc better, do you mind sending me a PR? I'd be happy to take a look at it. I'm not completely sure how this is failing on your end right now.
